bootstrap/utils/utils.go

```diff
@@ -161,3 +167,47 @@ func DeepCopy(src any, dest any) error {
 	}
 	return nil
 }
+
+func LoadFile(path string, contents *any) error {
```

Comment:
Must match signature of `os.ReadFile` which will be replaced with call to this function.

```suggestion
func LoadFile(path string) ([]byte, error) {
```